readme.md

```diff
@@ -81,7 +93,10 @@ var Panda = function(name, age) {
 ```
 
 Your Answer:
-```text
+```
+var eat_bamboo = function(){
+
+}
 
 ```
```

Comment:
If this method is to be applied to all Panda instances, we want to define it in the Panda prototype, like so...

```js
Panda.prototype.eat_bamboo = function() {
  this.num_bamboo_eaten = this.num_bamboo_eaten + 1;
}
```